packages/ra-ui-materialui/src/field/BooleanField.tsx

```diff
@@ -16,10 +17,16 @@ interface Props extends FieldProps {
     valueLabelFalse?: string;
 }
 
+const useStyles = makeStyles({
+    root: {
+        display: 'flex',
+    },
+});
```

Comment:
You should add `{ name: 'RaBooleanField' }`

Reply:
Absolutely, thanks for the review!